package-lock.json

```diff
@@ -1,504 +1,806 @@
 {
```

Comment:
can we remove package-lock from git completely here to comply with BE rules please?